docs/UserGuide.md

```diff
@@ -304,6 +304,19 @@ Examples:
 - `export f/csv` exports the contact list into a csv file.
 - `export f/vcf` exports the contact list into a vcf file.
 
+### Changing the application's theme : `toggle`
+
+Changes the application theme from light to dark or from dark to light.
+
+Format: `toggle`
+
+- If application is currently in light mode, toggle command will set it to dark mode.
+- If application is currently in dark mode, toggle command will set it to light mode.
+
```

Comment:
If we are not saving the theme state when exit we should indicate it here 

Like say: "The application will not remember the last used theme and will always default to dark mode on launch."

Incase during PE people say why the application never save the state